pkgs/development/libraries/json-glib/default.nix

```diff
@@ -37,23 +64,29 @@ stdenv.mkDerivation rec {
     pkg-config
     gettext
     glib
+    gi-docgen
+    libxslt
+    (if (stdenv.buildPlatform == stdenv.hostPlatform) then gobject-introspection else gobject-introspection-wrapper)
```

Comment:
If that is something universal we need everywhere we should move it into top-level or to the cross compile specific overlays.